gsoc/suborg_guidelines.md

```diff
@@ -13,20 +13,22 @@ slots we will get till the student selection process ends. However,
 it's our intention to keep the process fair for the suborganisations
 and students.
 
-## I want to have to become a sub-org admin, what do I do?
+## I want to become a sub-org admin, what do I do?
```

Comment:
is it sub-org admin though, or rather about a new sub-org?

Reply:
Let's go for "new sub-org"?